### DISCUSSION TITLE
Collection folders as route

### DISCUSSION THREAD

**Discussion by Syiana** [directus/directus#15772] [Archive]
Hey,

Directus is awesome, thanks for the great work!

I just miss some basic options like:

Is there a way to structure the data models?

Example: 
 
Folder App1
  -  Articles
  -  News

get: /items/app1/articles

Folder App2
  -  Articles
  -  News

get: /items/app2/articles

It would be awesome because then we could use Directus for multiple pages.

**Comment by azrikahar** [comment]
I think one "roadblock" for this structure is it's not possible to have 2 collections (or more precisely, 2 database tables) of the same name, as your example would require 2 `articles` tables and 2 `news` tables.

It should technically be possible to create a custom endpoint extension to do `/:folder/:collection` (eg. `/app1/articles1`, `/app2/articles2`) for a structure such as this:

- app1 (folder)
    - articles1
- app2 (folder)
    - articles2
    
but then it would be more straightforward to use `/items/articles1` and `/items/articles2` instead.

Perhaps an alternative structure would be to create an "app" collection (for `app1` and `app2), and create relationships between the app collection to articles and news, so that it will be much easier to filter the results based on the app.
